### PR TITLE
Staff Officer Vendors + Gormanization

### PR DESCRIPTION
**Pull request by Misttyy**
# About the pull request

This PR is meant to go along with my other PR adding a new cap for SOs in order to revitalize the role a little bit.

I've fixed a few bugs within this PR, mainly SOs could not access their own vendor before, and secondarily the CiC uniform vendor was improperly setup and did not respond to CiC access. Mapping also included adding an alternative uniform setup in SO/XO locker of officer uniform, gloves and external webbing, as we see Scott Gorman wear in the movie.

Besides that, I added a SO vendor that has gear to help them deploy, but lacks any gear that'd allow an SO to deploy completely solo like armor or primary weapons. It simply has some pistols, pouches, belts, and that sort.

# Explain why it's good for the game

This adds a vendor to give SOs a little bit more autonomy but that still requires them to get permission from their higher ups (XO and CO) to deploy. It makes their deployment process a little bit less tedious, fix some bugs and adds a new uniform. Hopefully, this'll encourage people to try the SO role out more.

# Testing Photographs and Procedure
<details>
<summary>Screenshots & Videos</summary>

image

CiC vendor set to command access now, not squad access.

image
Testing items and uniform from new uniform locker.

image
Testing vendor.

All done on local.

</details>

# Changelog

:cl: Mistfrag
add: Added external webbing, officer uniform and gloves to SO locker, alongside webbing to XOs locker in order to give them some more uniform choices.
add: Added a vendor for SOs. It includes a lengthy list of belts, pouches and spare supplies incase they're deploying too assist marines. For shipside SOs, it has some sidearms and a M3-VL vest.
del: Removed old things
fix: Fixed CiC uniform vendor being set to squad access. It now responds to CiC access.
fix: Fixed staff officers being unable to access their own vendor.
/:cl:

### CODE REVIEW

**Comment by Diegoflores31** [comment]
>add: Added a vendor for SOs. It includes a lengthy list of belts, pouches and spare supplies incase they're deploying too assist marines. For shipside SOs, it has some sidearms and a M3-VL vest

Arent SO Supposed to use the Officer armor ?

**Comment by Misttyy** [comment]
> > add: Added a vendor for SOs. It includes a lengthy list of belts, pouches and spare supplies incase they're deploying too assist marines. For shipside SOs, it has some sidearms and a M3-VL vest
> 
> Arent SO Supposed to use the Officer armor ?

Intentionally left proper armor out so they can't boonk gang equipment from their vendor and dip planet side to frag.

Same with why only sidearms appear for weapons.

File: code/game/machinery/vending/vendor_types/crew/staff_officer.dm [hunk 1]
```diff
@@ -1,5 +1,154 @@
...
+
+/obj/structure/machinery/cm_vending/clothing/staff_officer/get_listed_products(var/mob/user)
+	return GLOB.cm_vending_clothing_staff_officer
+
+//------------GEAR---------------
+
+GLOBAL_LIST_INIT(cm_vending_clothing_staff_officer, list(
+		list("STANDARD EQUIPMENT (TAKE ALL)", 0, null, null, null),
+		list("Boots", 0, /obj/item/clothing/shoes/marine/knife, MARINE_CAN_BUY_SHOES, VENDOR_ITEM_MANDATORY),
+		list("M3-VL Light Vest", 0, /obj/item/clothing/suit/storage/marine/light/vest, MARINE_CAN_BUY_ARMOR, VENDOR_ITEM_MANDATORY),
```
Comment (stanalbatross) [inline code/game/machinery/vending/vendor_types/crew/staff_officer.dm:15]: this is combat equipment (armour that it makes no sense to be mandatory)

File: code/game/machinery/vending/vendor_types/crew/staff_officer.dm [hunk 2]
```diff
@@ -1,5 +1,154 @@
...
+		list("Helmet", 0, /obj/item/clothing/head/helmet/marine/MP/SO, MARINE_CAN_BUY_HELMET, VENDOR_ITEM_MANDATORY),
+		list("MRE", 0, /obj/item/storage/box/MRE, MARINE_CAN_BUY_MRE, VENDOR_ITEM_MANDATORY),
+
+		list("PERSONAL SIDEARM (CHOOSE 1)", 0, null, null, null),
+		list("M44 Revolver", 0, /obj/item/storage/belt/gun/m44/mp, MARINE_CAN_BUY_BELT, VENDOR_ITEM_RECOMMENDED),
+		list("M4A3 Pistol", 0, /obj/item/storage/belt/gun/m4a3/commander, MARINE_CAN_BUY_BELT, VENDOR_ITEM_RECOMMENDED),
+		list("VP78 PDW", 0, /obj/item/storage/belt/gun/m4a3/vp78, MARINE_CAN_BUY_BELT, VENDOR_ITEM_RECOMMENDED),
+
+
+
```
Comment (stanalbatross) [inline code/game/machinery/vending/vendor_types/crew/staff_officer.dm:26]:
```suggestion
```

File: code/game/machinery/vending/vendor_types/crew/staff_officer.dm [hunk 3]
```diff
@@ -1,5 +1,154 @@
...
+		list("BACKPACK (CHOOSE 1)", 0, null, null, null),
+		list("Backpack", 0, /obj/item/storage/backpack/marine, MARINE_CAN_BUY_BACKPACK, VENDOR_ITEM_REGULAR),
+		list("Satchel", 0, /obj/item/storage/backpack/marine/satchel, MARINE_CAN_BUY_BACKPACK, VENDOR_ITEM_RECOMMENDED),
+
+		list("BELT (CHOOSE 1)", 0, null, null, null),
+		list("G8-A General Utility Pouch", 0, /obj/item/storage/backpack/general_belt, MARINE_CAN_BUY_BELT, VENDOR_ITEM_RECOMMENDED),
+		list("M276 Ammo Load Rig", 0, /obj/item/storage/belt/marine, MARINE_CAN_BUY_BELT, VENDOR_ITEM_RECOMMENDED),
+		list("M276 General Pistol Holster Rig", 0, /obj/item/storage/belt/gun/m4a3, MARINE_CAN_BUY_BELT, VENDOR_ITEM_REGULAR),
```
Comment (stanalbatross) [inline code/game/machinery/vending/vendor_types/crew/staff_officer.dm:34]: noobtrap remove

File: code/game/machinery/vending/vendor_types/crew/staff_officer.dm [hunk 4]
```diff
@@ -1,5 +1,154 @@
...
+		list("M276 Lifesaver Bag", 0, /obj/item/storage/belt/medical/lifesaver, MARINE_CAN_BUY_BELT, VENDOR_ITEM_REGULAR),
+		list("M276 Medical Storage Rig", 0, /obj/item/storage/belt/medical, MARINE_CAN_BUY_BELT, VENDOR_ITEM_REGULAR),
+		list("M276 M39 Holster Rig", 0, /obj/item/storage/large_holster/m39, MARINE_CAN_BUY_BELT, VENDOR_ITEM_REGULAR),
+		list("M276 M44 Holster Rig", 0, /obj/item/storage/belt/gun/m44, MARINE_CAN_BUY_BELT, VENDOR_ITEM_REGULAR),
```
Comment (stanalbatross) [inline code/game/machinery/vending/vendor_types/crew/staff_officer.dm:38]: noobtrap remove

File: code/game/machinery/vending/vendor_types/crew/staff_officer.dm [hunk 5]
```diff
@@ -1,5 +1,154 @@
...
+		list("Document Pouch", 0, /obj/item/storage/pouch/document, (MARINE_CAN_BUY_R_POUCH|MARINE_CAN_BUY_L_POUCH), VENDOR_ITEM_REGULAR),
+		list("Electronics Pouch (Full)", 0, /obj/item/storage/pouch/electronics/full, (MARINE_CAN_BUY_R_POUCH|MARINE_CAN_BUY_L_POUCH), VENDOR_ITEM_REGULAR),
+		list("First-Aid Pouch (Refillable Injectors)", 0, /obj/item/storage/pouch/firstaid/full, (MARINE_CAN_BUY_R_POUCH|MARINE_CAN_BUY_L_POUCH), VENDOR_ITEM_REGULAR),
+		list("First-Aid Pouch (Splints, Gauze, Ointment)", 0, /obj/item/storage/pouch/firstaid/full/alternate, (MARINE_CAN_BUY_R_POUCH|MARINE_CAN_BUY_L_POUCH), VENDOR_ITEM_REGULAR),
+		list("First-Aid Pouch (Pill Packets)", 0, /obj/item/storage/pouch/firstaid/full/pills, (MARINE_CAN_BUY_R_POUCH|MARINE_CAN_BUY_L_POUCH), VENDOR_ITEM_REGULAR),
+		list("First Responder Pouch", 0, /obj/item/storage/pouch/first_responder, (MARINE_CAN_BUY_R_POUCH|MARINE_CAN_BUY_L_POUCH), VENDOR_ITEM_REGULAR),
+		list("Flare Pouch (Full)", 0, /obj/item/storage/pouch/flare/full, (MARINE_CAN_BUY_R_POUCH|MARINE_CAN_BUY_L_POUCH), VENDOR_ITEM_REGULAR),
```
Comment (stanalbatross) [inline code/game/machinery/vending/vendor_types/crew/staff_officer.dm:61]: duped flare pouch

File: code/game/machinery/vending/vendor_types/crew/staff_officer.dm [hunk 6]
```diff
@@ -1,5 +1,154 @@
...
+		list("M276 Medical Storage Rig", 0, /obj/item/storage/belt/medical, MARINE_CAN_BUY_BELT, VENDOR_ITEM_REGULAR),
+		list("M276 M39 Holster Rig", 0, /obj/item/storage/large_holster/m39, MARINE_CAN_BUY_BELT, VENDOR_ITEM_REGULAR),
+		list("M276 M44 Holster Rig", 0, /obj/item/storage/belt/gun/m44, MARINE_CAN_BUY_BELT, VENDOR_ITEM_REGULAR),
+		list("M276 M82F Holster Rig", 0, /obj/item/storage/belt/gun/flaregun, MARINE_CAN_BUY_BELT, VENDOR_ITEM_REGULAR),
+		list("M276 Shotgun Shell Loading Rig", 0, /obj/item/storage/belt/shotgun, MARINE_CAN_BUY_BELT, VENDOR_ITEM_REGULAR),
+		list("M276 Toolbelt Rig (Full)", 0, /obj/item/storage/belt/utility/full, MARINE_CAN_BUY_BELT, VENDOR_ITEM_REGULAR),
+		list("M276 M40 Grenade Rig", 0, /obj/item/storage/belt/grenade, MARINE_CAN_BUY_BELT, VENDOR_ITEM_REGULAR),
+
+		list("POUCHES (CHOOSE 2)", 0, null, null, null),
+		list("Flare Pouch (Full)", 0, /obj/item/storage/pouch/flare/full, (MARINE_CAN_BUY_R_POUCH|MARINE_CAN_BUY_L_POUCH), VENDOR_ITEM_REGULAR),
```
Comment (stanalbatross) [inline code/game/machinery/vending/vendor_types/crew/staff_officer.dm:45]: duped

File: code/game/machinery/vending/vendor_types/crew/staff_officer.dm [hunk 7]
```diff
@@ -1,5 +1,154 @@
...
+		list("M276 Lifesaver Bag", 0, /obj/item/storage/belt/medical/lifesaver, MARINE_CAN_BUY_BELT, VENDOR_ITEM_REGULAR),
+		list("M276 Medical Storage Rig", 0, /obj/item/storage/belt/medical, MARINE_CAN_BUY_BELT, VENDOR_ITEM_REGULAR),
+		list("M276 M39 Holster Rig", 0, /obj/item/storage/large_holster/m39, MARINE_CAN_BUY_BELT, VENDOR_ITEM_REGULAR),
+		list("M276 M44 Holster Rig", 0, /obj/item/storage/belt/gun/m44, MARINE_CAN_BUY_BELT, VENDOR_ITEM_REGULAR),
+		list("M276 M82F Holster Rig", 0, /obj/item/storage/belt/gun/flaregun, MARINE_CAN_BUY_BELT, VENDOR_ITEM_REGULAR),
+		list("M276 Shotgun Shell Loading Rig", 0, /obj/item/storage/belt/shotgun, MARINE_CAN_BUY_BELT, VENDOR_ITEM_REGULAR),
+		list("M276 Toolbelt Rig (Full)", 0, /obj/item/storage/belt/utility/full, MARINE_CAN_BUY_BELT, VENDOR_ITEM_REGULAR),
+		list("M276 M40 Grenade Rig", 0, /obj/item/storage/belt/grenade, MARINE_CAN_BUY_BELT, VENDOR_ITEM_REGULAR),
+
+		list("POUCHES (CHOOSE 2)", 0, null, null, null),
```
Comment (stanalbatross) [inline code/game/machinery/vending/vendor_types/crew/staff_officer.dm:44]: this all needs to be in alphabetical order

File: code/game/machinery/vending/vendor_types/crew/staff_officer.dm [hunk 8]
```diff
@@ -1,5 +1,154 @@
...
+		list("First-Aid Pouch (Refillable Injectors)", 0, /obj/item/storage/pouch/firstaid/full, (MARINE_CAN_BUY_R_POUCH|MARINE_CAN_BUY_L_POUCH), VENDOR_ITEM_REGULAR),
+		list("First-Aid Pouch (Splints, Gauze, Ointment)", 0, /obj/item/storage/pouch/firstaid/full/alternate, (MARINE_CAN_BUY_R_POUCH|MARINE_CAN_BUY_L_POUCH), VENDOR_ITEM_REGULAR),
+		list("First-Aid Pouch (Pill Packets)", 0, /obj/item/storage/pouch/firstaid/full/pills, (MARINE_CAN_BUY_R_POUCH|MARINE_CAN_BUY_L_POUCH), VENDOR_ITEM_REGULAR),
+		list("First Responder Pouch", 0, /obj/item/storage/pouch/first_responder, (MARINE_CAN_BUY_R_POUCH|MARINE_CAN_BUY_L_POUCH), VENDOR_ITEM_REGULAR),
+		list("Flare Pouch (Full)", 0, /obj/item/storage/pouch/flare/full, (MARINE_CAN_BUY_R_POUCH|MARINE_CAN_BUY_L_POUCH), VENDOR_ITEM_REGULAR),
+		list("Large General Pouch", 0, /obj/item/storage/pouch/general/large, (MARINE_CAN_BUY_R_POUCH|MARINE_CAN_BUY_L_POUCH), VENDOR_ITEM_REGULAR),
+		list("Sling Pouch", 0, /obj/item/storage/pouch/sling, (MARINE_CAN_BUY_R_POUCH|MARINE_CAN_BUY_L_POUCH), VENDOR_ITEM_REGULAR),
```
Comment (stanalbatross) [inline code/game/machinery/vending/vendor_types/crew/staff_officer.dm:63]: duped

File: code/game/machinery/vending/vendor_types/crew/staff_officer.dm [hunk 9]
```diff
@@ -1,5 +1,154 @@
...
+		list("Entrenching Tool", 2, /obj/item/tool/shovel/etool, null, VENDOR_ITEM_REGULAR),
+		list("High-Capacity Power Cell", 3, /obj/item/cell/high, null, VENDOR_ITEM_REGULAR),
+		list("Light Replacer", 2, /obj/item/device/lightreplacer, null, VENDOR_ITEM_REGULAR),
+		list("Metal x10", 5, /obj/item/stack/sheet/metal/small_stack, null, VENDOR_ITEM_REGULAR),
+		list("Multitool", 4, /obj/item/device/multitool, null, VENDOR_ITEM_REGULAR),
+		list("Plasteel x10", 7, /obj/item/stack/sheet/plasteel/small_stack, null, VENDOR_ITEM_REGULAR),
+		list("Sandbags x25", 10, /obj/item/stack/sandbags_empty/half, null, VENDOR_ITEM_REGULAR),
+		list("Industrial Blowtorch", 2, /obj/item/tool/weldingtool/largetank, null, VENDOR_ITEM_REGULAR),
+		list("High-Capacity Industrial Blowtorch", 4, /obj/item/tool/weldingtool/hugetank, null, VENDOR_ITEM_REGULAR),
+		list("Plastic Explosive", 3, /obj/item/explosive/plastic, null, VENDOR_ITEM_REGULAR),
```
Comment (stanalbatross) [inline code/game/machinery/vending/vendor_types/crew/staff_officer.dm:96]: they don't have the engineering skill to use half of this why have you just copypasted it in here

File: code/game/machinery/vending/vendor_types/crew/staff_officer.dm [hunk 10]
```diff
@@ -1,5 +1,154 @@
...
+
+		list("FIRSTAID KITS", 0, null, null, null),
```
Comment (stanalbatross) [inline code/game/machinery/vending/vendor_types/crew/staff_officer.dm:98]: all this stuff that you've copypasted from medic vendors just massively streamlines deployment, not desirable, remove

File: code/game/machinery/vending/vendor_types/crew/staff_officer.dm [hunk 11]
```diff
@@ -1,5 +1,154 @@
...
+		list("ENGINEER SUPPLIES", 0, null, null, null),
+		list("Airlock Circuit Board", 2, /obj/item/circuitboard/airlock, null, VENDOR_ITEM_REGULAR),
+		list("APC Circuit Board", 2, /obj/item/circuitboard/apc, null, VENDOR_ITEM_REGULAR),
+		list("Entrenching Tool", 2, /obj/item/tool/shovel/etool, null, VENDOR_ITEM_REGULAR),
+		list("High-Capacity Power Cell", 3, /obj/item/cell/high, null, VENDOR_ITEM_REGULAR),
+		list("Light Replacer", 2, /obj/item/device/lightreplacer, null, VENDOR_ITEM_REGULAR),
+		list("Metal x10", 5, /obj/item/stack/sheet/metal/small_stack, null, VENDOR_ITEM_REGULAR),
+		list("Multitool", 4, /obj/item/device/multitool, null, VENDOR_ITEM_REGULAR),
+		list("Plasteel x10", 7, /obj/item/stack/sheet/plasteel/small_stack, null, VENDOR_ITEM_REGULAR),
+		list("Sandbags x25", 10, /obj/item/stack/sandbags_empty/half, null, VENDOR_ITEM_REGULAR),
```
Comment (stanalbatross) [inline code/game/machinery/vending/vendor_types/crew/staff_officer.dm:93]: remove mats

File: code/game/machinery/vending/vendor_types/crew/staff_officer.dm [hunk 12]
```diff
@@ -1,5 +1,154 @@
...
+		list("Industrial Blowtorch", 2, /obj/item/tool/weldingtool/largetank, null, VENDOR_ITEM_REGULAR),
+		list("High-Capacity Industrial Blowtorch", 4, /obj/item/tool/weldingtool/hugetank, null, VENDOR_ITEM_REGULAR),
```
Comment (stanalbatross) [inline code/game/machinery/vending/vendor_types/crew/staff_officer.dm:95]: remove tools, too much streamlining

File: code/game/machinery/vending/vendor_types/crew/staff_officer.dm [hunk 13]
```diff
@@ -1,5 +1,154 @@
...
+		list("Medkit Pouch", 0, /obj/item/storage/pouch/medkit, (MARINE_CAN_BUY_R_POUCH|MARINE_CAN_BUY_L_POUCH), VENDOR_ITEM_REGULAR),
+		list("Pressurized Reagent Canister Pouch (Bicaridine)", 0, /obj/item/storage/pouch/pressurized_reagent_canister/bicaridine, (MARINE_CAN_BUY_R_POUCH|MARINE_CAN_BUY_L_POUCH), VENDOR_ITEM_REGULAR),
+		list("Pressurized Reagent Canister Pouch (Kelotane)", 0, /obj/item/storage/pouch/pressurized_reagent_canister/kelotane, (MARINE_CAN_BUY_R_POUCH|MARINE_CAN_BUY_L_POUCH), VENDOR_ITEM_REGULAR),
+		list("Pressurized Reagent Canister Pouch (Revival Mix)", 0, /obj/item/storage/pouch/pressurized_reagent_canister/revival, (MARINE_CAN_BUY_R_POUCH|MARINE_CAN_BUY_L_POUCH), VENDOR_ITEM_REGULAR),
+		list("Pressurized Reagent Canister Pouch (Tricordrazine)", 0, /obj/item/storage/pouch/pressurized_reagent_canister/tricordrazine, (MARINE_CAN_BUY_R_POUCH|MARINE_CAN_BUY_L_POUCH), VENDOR_ITEM_REGULAR),
+		list("Pressurized Reagent Canister Pouch (EMPTY)", 0, /obj/item/storage/pouch/pressurized_reagent_canister, (MARINE_CAN_BUY_R_POUCH|MARINE_CAN_BUY_L_POUCH), VENDOR_ITEM_REGULAR),
```
Comment (stanalbatross) [inline code/game/machinery/vending/vendor_types/crew/staff_officer.dm:70]: too much streamlining if they want one go to research

File: maps/map_files/USS_Almayer/USS_Almayer.dmm [hunk 14]
```diff
@@ -25636,6 +25643,12 @@
 	dir = 8
 	},
 /obj/structure/closet/secure_closet/staff_officer/gear,
+/obj/item/clothing/under/marine/officer/command,
+/obj/item/clothing/under/marine/officer/command,
+/obj/item/clothing/suit/storage/webbing,
+/obj/item/clothing/suit/storage/webbing,
+/obj/item/clothing/gloves/combat,
+/obj/item/clothing/gloves/combat,
```
Comment (stanalbatross) [inline maps/map_files/USS_Almayer/USS_Almayer.dmm:25651]: just edit the locker in code no need for these

File: maps/map_files/USS_Almayer/USS_Almayer.dmm [hunk 15]
```diff
@@ -65476,6 +65489,18 @@
 	tag = "icon-sterile_green_side (NORTH)"
 	},
 /area/almayer/medical/medical_science)
+"pYq" = (
+/obj/structure/machinery/cm_vending/clothing/staff_officer{
+	density = 0;
+	pixel_x = -30;
+	desc = "An automated equipment vendor for Staff Officers.";
+	name = "\improper ColMarTech Staff Officer Equipment Rack"
```
Comment (stanalbatross) [inline maps/map_files/USS_Almayer/USS_Almayer.dmm:65497]: put it on the object

File: maps/map_files/USS_Almayer/USS_Almayer.dmm [hunk 16]
```diff
@@ -68809,7 +68834,9 @@
 	},
 /area/almayer/engineering/upper_engineering)
 "ryR" = (
-/obj/structure/machinery/cm_vending/sorted/uniform_supply/squad_prep,
+/obj/structure/machinery/cm_vending/sorted/uniform_supply/squad_prep{
+	req_access = list(1)
```
Comment (stanalbatross) [inline maps/map_files/USS_Almayer/USS_Almayer.dmm:68838]: what is going on here

**Review by stanalbatross — CHANGES_REQUESTED** [review]

File: maps/map_files/USS_Almayer/USS_Almayer.dmm (hunk 16 above)
Comment (Misttyy) [reply maps/map_files/USS_Almayer/USS_Almayer.dmm:68838]: It doesn't unless you've had different experiences testing on local. It worked for SOs when I tested it.

File: code/game/machinery/vending/vendor_types/crew/staff_officer.dm (hunk 1 above)
Comment (Misttyy) [reply code/game/machinery/vending/vendor_types/crew/staff_officer.dm:15]: are you saying to remove it or remove it as mandatory

**Comment by Misttyy** [comment]
Should be good for merging now, @stanalbatross

File: code/game/machinery/vending/vendor_types/crew/staff_officer.dm [hunk 17]
```diff
@@ -1,5 +1,127 @@
...
+		list("STANDARD EQUIPMENT (TAKE ALL)", 0, null, null, null),
+		list("Boots", 0, /obj/item/clothing/shoes/marine/knife, MARINE_CAN_BUY_SHOES, VENDOR_ITEM_MANDATORY),,
+		list("Headset", 0, /obj/item/device/radio/headset/almayer/mcom, MARINE_CAN_BUY_EAR, VENDOR_ITEM_MANDATORY),
+		list("Helmet", 0, /obj/item/clothing/head/helmet/marine/MP/SO, MARINE_CAN_BUY_HELMET, VENDOR_ITEM_MANDATORY),
+		list("MRE", 0, /obj/item/storage/box/MRE, MARINE_CAN_BUY_MRE, VENDOR_ITEM_MANDATORY),
+
+		list("PERSONAL SIDEARM (CHOOSE 1)", 0, null, null, null),
+		list("M44 Revolver", 0, /obj/item/storage/belt/gun/m44/mp, MARINE_CAN_BUY_BELT, VENDOR_ITEM_RECOMMENDED),
+		list("M4A3 Pistol", 0, /obj/item/storage/belt/gun/m4a3/commander, MARINE_CAN_BUY_BELT, VENDOR_ITEM_RECOMMENDED),
+		list("VP78 PDW", 0, /obj/item/storage/belt/gun/m4a3/vp78, MARINE_CAN_BUY_BELT, VENDOR_ITEM_RECOMMENDED),
```
Comment (stanalbatross) [inline code/game/machinery/vending/vendor_types/crew/staff_officer.dm:22]: this isn't called a pdw anywhere else, change it we hate inconsistent names

File: code/game/machinery/vending/vendor_types/crew/staff_officer.dm [hunk 18]
```diff
@@ -1,5 +1,127 @@
...
+		list("Electronics Pouch (Full)", 0, /obj/item/storage/pouch/electronics/full, (MARINE_CAN_BUY_R_POUCH|MARINE_CAN_BUY_L_POUCH), VENDOR_ITEM_REGULAR),
+		list("Flare Pouch (Full)", 0, /obj/item/storage/pouch/flare/full, (MARINE_CAN_BUY_R_POUCH|MARINE_CAN_BUY_L_POUCH), VENDOR_ITEM_REGULAR),
+		list("Fuel Tank Strap Pouch", 0, /obj/item/storage/pouch/flamertank, (MARINE_CAN_BUY_R_POUCH|MARINE_CAN_BUY_L_POUCH), VENDOR_ITEM_REGULAR),
+		list("First-Aid Pouch (Refillable Injectors)", 0, /obj/item/storage/pouch/firstaid/full, (MARINE_CAN_BUY_R_POUCH|MARINE_CAN_BUY_L_POUCH), VENDOR_ITEM_REGULAR),
+		list("First-Aid Pouch (Splints, Gauze, Ointment)", 0, /obj/item/storage/pouch/firstaid/full/alternate, (MARINE_CAN_BUY_R_POUCH|MARINE_CAN_BUY_L_POUCH), VENDOR_ITEM_REGULAR),
+		list("First-Aid Pouch (Pill Packets)", 0, /obj/item/storage/pouch/firstaid/full/pills, (MARINE_CAN_BUY_R_POUCH|MARINE_CAN_BUY_L_POUCH), VENDOR_ITEM_REGULAR),
+		list("First Responder Pouch", 0, /obj/item/storage/pouch/first_responder, (MARINE_CAN_BUY_R_POUCH|MARINE_CAN_BUY_L_POUCH), VENDOR_ITEM_REGULAR),
```
Comment (stanalbatross) [inline code/game/machinery/vending/vendor_types/crew/staff_officer.dm:48]: i comes before u and l

File: code/game/machinery/vending/vendor_types/crew/staff_officer.dm [hunk 19]
```diff
@@ -1,5 +1,127 @@
...
+		list("Large Magazine Pouch", 0, /obj/item/storage/pouch/magazine/large, (MARINE_CAN_BUY_R_POUCH|MARINE_CAN_BUY_L_POUCH), VENDOR_ITEM_REGULAR),
+		list("Large Shotgun Shell Pouch", 0, /obj/item/storage/pouch/shotgun/large, (MARINE_CAN_BUY_R_POUCH|MARINE_CAN_BUY_L_POUCH), VENDOR_ITEM_REGULAR),
+		list("Large Pistol Magazine Pouch", 0, /obj/item/storage/pouch/magazine/pistol/large, (MARINE_CAN_BUY_R_POUCH|MARINE_CAN_BUY_L_POUCH), VENDOR_ITEM_REGULAR),
+		list("Medical Pouch", 0, /obj/item/storage/pouch/medical, (MARINE_CAN_BUY_R_POUCH|MARINE_CAN_BUY_L_POUCH), VENDOR_ITEM_REGULAR),
+		list("Medkit Pouch", 0, /obj/item/storage/pouch/medkit, (MARINE_CAN_BUY_R_POUCH|MARINE_CAN_BUY_L_POUCH), VENDOR_ITEM_REGULAR),
+		list("Pistol Pouch", 0, /obj/item/storage/pouch/pistol, (MARINE_CAN_BUY_R_POUCH|MARINE_CAN_BUY_L_POUCH), VENDOR_ITEM_REGULAR),
+		list("Sling Pouch", 0, /obj/item/storage/pouch/sling, (MARINE_CAN_BUY_R_POUCH|MARINE_CAN_BUY_L_POUCH), VENDOR_ITEM_REGULAR),
+		list("Sling Pouch", 0, /obj/item/storage/pouch/sling, (MARINE_CAN_BUY_R_POUCH|MARINE_CAN_BUY_L_POUCH), VENDOR_ITEM_REGULAR),
```
Comment (stanalbatross) [inline code/game/machinery/vending/vendor_types/crew/staff_officer.dm:57]: ??? duped

File: code/game/machinery/vending/vendor_types/crew/staff_officer.dm [hunk 20]
```diff
@@ -1,5 +1,127 @@
...
+		list("Tools Pouch (Full)", 0, /obj/item/storage/pouch/tools/full, (MARINE_CAN_BUY_R_POUCH|MARINE_CAN_BUY_L_POUCH), VENDOR_ITEM_REGULAR),
+
+		list("ACCESSORIES (CHOOSE 1)", 0, null, null, null),
+		list("Black Webbing Vest", 0, /obj/item/clothing/accessory/storage/black_vest, MARINE_CAN_BUY_ACCESSORY, VENDOR_ITEM_REGULAR),
+		list("Brown Webbing Vest", 0, /obj/item/clothing/accessory/storage/black_vest/brown_vest, MARINE_CAN_BUY_ACCESSORY, VENDOR_ITEM_RECOMMENDED),
+		list("Shoulder Holster", 0, /obj/item/clothing/accessory/storage/holster, MARINE_CAN_BUY_ACCESSORY, VENDOR_ITEM_REGULAR),
+		list("Webbing", 0, /obj/item/clothing/accessory/storage/webbing, MARINE_CAN_BUY_ACCESSORY, VENDOR_ITEM_REGULAR),
+		list("Drop Pouch", 0, /obj/item/clothing/accessory/storage/droppouch, MARINE_CAN_BUY_ACCESSORY, VENDOR_ITEM_REGULAR),
```
Comment (stanalbatross) [inline code/game/machinery/vending/vendor_types/crew/staff_officer.dm:65]: alphabetical order?

File: code/game/machinery/vending/vendor_types/crew/staff_officer.dm (hunk 10 above)
Comment (stanalbatross) [reply code/game/machinery/vending/vendor_types/crew/staff_officer.dm:71]: hello

File: maps/map_files/USS_Almayer/USS_Almayer.dmm [hunk 21]
```diff
@@ -65476,6 +65476,18 @@
 	tag = "icon-sterile_green_side (NORTH)"
 	},
 /area/almayer/medical/medical_science)
+"pYq" = (
+/obj/structure/machinery/cm_vending/clothing/staff_officer{
+	density = 0;
+	pixel_x = -30;
+	desc = "An automated equipment vendor for Staff Officers.";
+	name = "\improper ColMarTech Staff Officer Equipment Rack"
```
Comment (stanalbatross) [inline maps/map_files/USS_Almayer/USS_Almayer.dmm:65484]: ????? WHY

**Review by stanalbatross — CHANGES_REQUESTED** [review]

**Review by stanalbatross — APPROVED** [review]

**Pull request merged**